### DISCUSSION TITLE
osu!lazer cannot change game resolution to fullscreen

### DISCUSSION THREAD

**Discussion by KosiarekOfficial** [ppy/osu#21635] [Q&A] [answered]
Hi
I've been testing osu!lazer for couple days (for more time) and today I checked my game settings and I saw that my game is not working in fullscreen when I choosed the fullscreen option. It says: (translated from polish) "Failed to enter fullscreen mode. Some delays from input data may be felt" and I really don't know why I have this. I have installed osu!lazer a couple months ago and it's up to date. In stable version my game works perfectly on fullscreen and I want to repair in lazer, because sometimes I have lags that spoils the fun of the game sometimes.
Tell me if I need to send any screenshots, information about my laptop (because I'm playing on laptop, MSI GL72M 7RDX with Win 11)

**Comment by peppy** [comment, marked as answer]
We are working on a new graphics backend which will fix this in the near future. Please wait for that!
